src/chains/supported-chains-list.ts

```diff
@@ -1,57 +1,75 @@
 export const CHAINS = {
   TESTNET: {
+    ACRE: "acre",
+    ARBITRUM: "arbitrum",
     AURA: "aura",
-    AXELAR: "axelarnet",
-    COMDEX: "comdex",
-    EVMOS: "evmos",
-    FETCH: "fetch",
-    KUJIRA: "kujira",
-    OSMOSIS: "osmosis-5",
-    SEI: "sei",
     AURORA: "aurora",
+    AXELAR: "Axelarnet",
     AVALANCHE: "Avalanche",
+    BASE: "base",
     BINANCE: "binance",
     BNBCHAIN: "binance",
+    BURNT: "burnt",
+    CELO: "celo",
+    COMDEX: "comdex-2",
     ETHEREUM: "ethereum-2",
+    EVMOS: "evmos",
     FANTOM: "Fantom",
+    FETCH: "fetch",
+    FILECOIN: "filecoin",
+    KAVA: "kava",
+    KUJIRA: "kujira",
     MOONBEAM: "Moonbeam",
+    NEUTRON: "neutron",
+    ODIN: "odin",
+    OPTIMISM: "optimism",
+    OSMOSIS: "osmosis-6",
+    PERSISTENCE: "persistence",
     POLYGON: "Polygon",
+    SECRETSNIP: "secret-snip",
+    SECRETSNIP2: "secret-snip-2",
```

Comment:
```suggestion
    SECRETSNIP: "secret-snip-2",
```
this is the official testnet one now